src/main/java/de/homemadeapps/manager/ItemManager.java

```diff
@@ -27,7 +27,7 @@ public boolean saveItem(final String name, final String description) {
         boolean doExist = false;
         final Item item = new Item();
         item.setName(name);
-        item.setName(description);
+        item.setDescription(description);
```

Comment:
Vielleicht noch einen Test hinzufügen, der dieses Verhalten auch sicherstellt? In den bisherigen Tests wird nur die Rückgabe geprüft, aber nicht, was save/update eigentlich mit der DB anstellt.

Reply:
Macht das testen sehr ekelhaft, weil ich dann nicht mehr richtig mocken kann :(